net/rs-aggregate/Portfile

```
@@ -0,0 +1,219 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        ktims rs-aggregate v0.3.2
```

The format of `github.setup` should be `<author> <project> <version> <prefix> <postfix>`

So this should be:

```suggestion
github.setup        ktims rs-aggregate 0.3.2 v
```

---

Thanks for the heads-up!